report/src/main/java/com/hartwig/actin/report/pdf/tables/clinical/LabResultsGenerator.java

```diff
@@ -64,7 +88,12 @@ public String title() {
     @NotNull
     @Override
     public Table contents() {
-        Set<LocalDate> dates = selectDates(labInterpretation.allDates());
+        List<LocalDate> dates = labInterpretation.allDates()
```

Comment:
The entries in this collection have to be unique. Might be enforced already by providing class, but wouldn't still be better to use a Set rather than a List here?

Reply:
They're provided as a set, so uniqueness is guaranteed. We could enforce it in the stream to be safe though.

I wanted to use a list to preserve order. With a set, there's no guarantee that we'll use the same date order for each set of measurements.

Reply:
Clear and wasn't aware of that!